Tools/ardupilotwaf/boards.py

```diff
@@ -425,6 +425,8 @@ def get_ap_periph_boards():
     for d in dirlist:
         if d in list_ap:
             continue
+        if 'Here' in d:
```

Comment:
Oh, I don't like this....  let's not sprinkle product names through the actual code, please keep it in data as much as possible,

Reply:
it should find AP_PERIPH in the hwdef.dat, so this should not be needed?

Reply:
AP_Periph define is in the .inc hwdef not the .dat.

Reply:
We plan on having Here in the name of all our AP_Periph products, they will include HereCommon/hwdef.inc . So, most if not all the hwdef based on them will not have AP_Periph in the hwdef, that is why I had to make that change.